specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -12434,6 +12478,46 @@
           }
         }
       }
+    },
+    "TieringPolicy": {
+      "description": "Tiering Policy.",
+      "type": "object",
+      "properties": {
+        "tieringMode": {
+          "description": "Tiering Mode to control automatic tiering of recovery points",
+          "enum": [
+            "Invalid",
+            "TierRecommended",
+            "TierAfter",
+            "DoNotTier"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "TieringMode",
+            "modelAsString": true
+          }
+        },
+        "duration": {
+          "format": "int32",
+          "description": "Number of days/weeks/months/years to retain backups in current tier before tiering.",
+          "type": "integer"
+        },
+        "durationType": {
+          "description": "Retention duration type: days/weeks/months/years",
+          "enum": [
+            "Invalid",
```

Comment:
>Invalid [](http://example.com/codeflow?start=13&length=7)

When would this be used?

Reply:
In case where the tiering mode is TierRecommended

Reply:
Can you call out that that the duration values are only applicable when the enum is TierAfter?

Reply:
Added to the description